i18n_scripts/combine_translations_files.py

```diff
@@ -71,11 +71,10 @@ def _escape(s):
     replacement_function = lambda match: {
         '\n': r'\n',
         '\r': r'\r',
-        '\\': r'\\',
         '"': r'\"'
     }[match.group(0)]
 
-    s = re.sub(r'([\n\r\\"])', replacement_function, s)
+    s = re.sub(r'([\n\r"])', replacement_function, s)
```

Comment:
does it work on multiple characters in the same string?

`yes """" it's me`

Reply:
Yes, it does.